### PR TITLE
11/14/2024 Production Deploy

### PR DESCRIPTION
**Pull request by ccostino**
*A note to PR reviewers: it may be helpful to review our [code review documentation](https://github.com/GSA/notifications-api/blob/main/docs/all.md#code-reviews) to know what to keep in mind while reviewing pull requests.*

## Description

Please enter a detailed description here.

This is a production deploy for 11/14/2024.  [Full release notes are here!](https://github.com/GSA/notifications-admin/releases/tag/11-14-2024)

## Security Considerations

* All changes have been reviewed and tested prior to being included here.

### CODE REVIEW

File: app/config.py [hunk 1]
```diff
@@ -87,6 +87,9 @@ class Config(object):
             "tts-benefits-studio@gsa.gov",
         ],
     }
+    FEATURE_BEST_PRACTICES_ENABLED = (
+        getenv("FEATURE_BEST_PRACTICES_ENABLED", "false") == "true"
```
Comment (terrazoon) [inline app/config.py:91]: Probably doesn't matter, but this isn't standard python capitalization for bools

File: app/main/views/index.py [hunk 2]
```diff
@@ -1,44 +1,60 @@
...
+# Hook to check for guidance routes
+@main.before_request
+def check_guidance_feature():
+    current_app.logger.warning("best practices 1234")
+    current_app.logger.warning(current_app.config["FEATURE_BEST_PRACTICES_ENABLED"])
+    if (
+        request.path.startswith("/best-practices")
+        and not current_app.config["FEATURE_BEST_PRACTICES_ENABLED"]
+    ):
+        abort(404)
```
Comment (terrazoon) [inline app/main/views/index.py:30]: should these aborts log the reason we return 404?

File: app/templates/views/best-practices/clear-goals.html [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+        the end in mind.
+      </p>
+      <div class="usa-summary-box maxw-tablet" role="region" aria-labelledby="summary-box-establish-clear-goals">
+        <div class="usa-summary-box__body">
+          <p class="usa-summary-box__heading" id="summary-box-establish-clear-goals">
+            To establish clear goals, start by answering the following questions:
+          </p>
+          <div class="usa-summary-box__text">
+            <ul class="usa-list">
+              <li><i>Who</i> do you want to text?</li>
```
Comment (terrazoon) [inline app/templates/views/best-practices/clear-goals.html:29]: Whom. :)

File: app/main/views/index.py (hunk 2 above)
Comment (ccostino) [reply app/main/views/index.py:30]: Good catch for an improvement, yes!

**Review by terrazoon — APPROVED** [review]
approved with a couple comments

**Review by stvnrlly — APPROVED** [review]

**Review by xlorepdarkhelm — APPROVED** [review]
Same comments as Ken.

File: app/templates/views/best-practices/clear-goals.html (hunk 3 above)
Comment (ccostino) [reply app/templates/views/best-practices/clear-goals.html:29]: Noting in Slack for the future! 🙂

**Pull request merged**